src/main/java/it/gov/pagopa/payhub/activities/dto/classifications/IufClassificationActivityResult.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.payhub.activities.dto.classifications;
+
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class IufClassificationActivityResult {
+    private List<PaymentsReportingDTO> paymentsReportingDTOS;
```

Comment:
we don't need the entire PaymentsReportingDTO as result

Reply:
Adding dto containing only needed fields

public class ClassifyResultDTO {
	private Long organizationId;
	private String creditorReferenceId;
	private String regulationUniqueIdentifier;
	private int transferIndex;
}